MultiPartyExample/MultiPartyViewController.swift

```diff
@@ -0,0 +1,566 @@
+    }
+
+    override var prefersHomeIndicatorAutoHidden: Bool {
+        return room != nil
+    }
+
+    override func viewWillLayoutSubviews() {
+        super.viewWillLayoutSubviews()
+
+        var layoutFrame = self.view.bounds
+        if #available(iOS 11.0, *) {
+            // Ensure the preview fits in the safe area.
+            let safeAreaGuide = self.view.safeAreaLayoutGuide
+            layoutFrame = safeAreaGuide.layoutFrame
+        }
+
+        // Layout the message label.
+        messageLabel.preferredMaxLayoutWidth = layoutFrame.width - (MultiPartyViewController.kTextPadding * 4)
+        messageLabel.frame = CGRect(x: layoutFrame.minX + MultiPartyViewController.kTextPadding,
+                                    y: layoutFrame.minY + MultiPartyViewController.kTextPadding,
+                                    width: layoutFrame.width - (MultiPartyViewController.kTextPadding * 4),
+                                    height: 18).integral
+
+        let topY = messageLabel.frame.maxY
+        let totalHeight = layoutFrame.height - topY
+        let totalWidth = layoutFrame.width
+
+        let videoViewHeight = round(totalHeight / 2)
+        let videoViewWidth = round(totalWidth / 2)
+
+        let videoViewSize = CGSize(width: videoViewWidth, height: videoViewHeight)
+
+        // Layout local Participant
+        localParticipantVideoView.frame = CGRect(origin: CGPoint(x: layoutFrame.minX, y: topY),
+                                                 size: videoViewSize).insetBy(dx: 4, dy: 4)
+
+        let audioIndicatorSize = localParticipantAudioIndicator.intrinsicContentSize
+        let audioIndicatorOrigin = CGPoint(x: layoutFrame.minX + videoViewWidth - audioIndicatorSize.width - MultiPartyViewController.kAudioIndicatorPadding,
+                                           y: videoViewHeight + topY - audioIndicatorSize.height - MultiPartyViewController.kAudioIndicatorPadding)
+
+        // Layout remote Participants
+        localParticipantAudioIndicator.frame = CGRect(origin: audioIndicatorOrigin, size: audioIndicatorSize)
+
+        var index = 0
+        for remoteParticipantView in remoteParticipantViews {
+            switch index {
+            case 0:
+                remoteParticipantView.frame = CGRect.init(origin: CGPoint.init(x: layoutFrame.minX + videoViewWidth, y: topY),
+                                                          size: videoViewSize)
+            case 1:
+                remoteParticipantView.frame = CGRect.init(origin: CGPoint(x: layoutFrame.minX, y: topY + videoViewHeight),
+                                                          size: videoViewSize)
+            case 2:
+                remoteParticipantView.frame = CGRect.init(origin: CGPoint(x: layoutFrame.minX + videoViewWidth, y: topY + videoViewHeight),
+                                                          size: videoViewSize)
+            default:
+                break
+            }
+            index += 1
+        }
+    }
```

Comment:
There is lot of dynamic layout code here, I feel this can be moved in storyboard. I guess we will fix it in next iteration  when we modify the app.